app/src/main/java/mozilla/lockbox/support/FileReader.kt

```diff
@@ -0,0 +1,10 @@
+package mozilla.lockbox.support
+
+import java.io.FileReader
+
+class FileReader {
+    fun readContents(filePath: String): String {
+        val file = FileReader(filePath)
+        return file.readText()
```

Comment:
Erm. Ok. I'm not sure I understand why so much code here.

But we're not passing this into the store, so we can test it?

Reply:
mmm, mostly because I wanted to avoid testing our filesystem interactions inside of the autolockstore...

Reply:
as far as testing goes: yes, we can mock the new instances with PowerMock.